client/src/intl/en.json

```diff
@@ -187,6 +171,14 @@
     "defaultMessage": "Percentile (0-100)",
     "description": "the population of the feature selected"
   },
+  "banner.beta.info": {
+    "defaultMessage": "It is an early, in-progress version of the tool with limited data sets that will be continuously updated.",
```

Comment:
Is this the right message? It just seems weird that it's "It is an early..." as opposed to something like "This is an early..." (What is `it` referring to?)